postgres/functions/refresh_matviews.sql

```diff
@@ -16,3 +16,6 @@ BEGIN
   RETURN 1;
 END;
 $function$;
+
+-- Called in refresh_app_monitoring.sh
```

Comment:
can we actually add a cron file ? or put this in the readme?

Reply:
Added to readme